recipes/IWSLT22_lowresource/data_proc/prepare_tamasheq.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# ############################################################################
+# Tamasheq-French data processing pipeline
+#
+# Requirements: python, git, sentencepiece command line extension
+# Author:  Marcely Zanon Boito, 2022
+# ############################################################################
+
+cd data_proc/
+
+# 1. clone the IWSLT 2022 Tamasheq-French dataset
+git clone https://github.com/mzboito/IWSLT2022_Tamasheq_data.git
+
+# 2. train the tokenizer
+# /!\ it requires the command line extension for sentence piece, available here: https://github.com/google/sentencepiece
+spm_train --input IWSLT2022_Tamasheq_data/taq_fra_clean/train/txt/train.fra  --vocab_size=1000 --model_type=unigram --model_prefix=IWSLT2022_Tamasheq_data/taq_fra_clean/train/spm_unigram1000
```

Comment:
Hi ! Can't we make this compatible with SB? By this, I mean that we can already train a tokenizer with SB. It is not usual for a us to have a shell script, and in fact, we don't really need it. The whole tokenizer pre-training and data_preparation can be done within a single train.py file :-)